console_libraries/menu.lib

```diff
@@ -61,6 +61,19 @@
 {{ template "_menuItem" (args . "cassandra.html" "Cassandra") }}
 {{ end }}
 
+{{ if query "up{job=~'blackbox.*'}" }}
```

Comment:
OK, done. Seems a shame since it's way less functionality and it would scale up from one job smoothly. Not pushed yet, just making sure it works now.